agenta-cli/agenta/sdk/agenta_decorator.py

```diff
@@ -396,6 +397,11 @@ def find_in_schema(schema: dict, param_name: str, xparam: str):
         f"Body_{func_name}_{endpoint}_post"
     ]["properties"]
     for param_name, param_val in params.items():
+        if isinstance(param_val, GroupedMultipleChoiceParam):
+            subschema = find_in_schema(schema_to_override, param_name, "grouped_choice")
+            if subschema:
+                subschema["choices"] = param_val.options
```

Comment:
1. Why use a different name, let's use choices or options
2. Maybe add a small comment on the data type for clarity